code/controllers/configuration.dm

```diff
@@ -652,6 +653,9 @@ var/list/gamemode_cache = list()
 				if("tickcomp")
 					Tickcomp = 1
 
+				if("ipc_timelock_active")
+					ipc_timelock_active = FALSE
```

Comment:
if you uncomment this, and it is activated, should this not be turned to TRUE

Reply:
copypaste moment